LICENSE

```diff
@@ -11,7 +11,7 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution.
 
-* Neither the name of the {organization} nor the names of its
+* Neither the name of angular-g11n nor the names of its
```

Comment:
We're using the so-called "new", [BSD-3](http://choosealicense.com/licenses/bsd-3-clause/) license. In the template for the BSD-3, `the name of the {organization}` was changed to `the name of [project]`.